config/default/field.field.node.hs_private_page.field_hs_priv_page_components.yml

```diff
@@ -28,79 +28,44 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
-      hs_priv_text_area: hs_priv_text_area
       hs_accordion: hs_accordion
-      hs_clr_bnd: hs_clr_bnd
       hs_postcard: hs_postcard
+      hs_priv_text_area: hs_priv_text_area
       hs_priv_collection: hs_priv_collection
       hs_sptlght_slder: hs_sptlght_slder
-      hs_timeline: hs_timeline
     negate: 0
     target_bundles_drag_drop:
       hs_accordion:
-        weight: -44
+        weight: 9
         enabled: true
       hs_banner:
-        weight: -36
-        enabled: false
-      hs_callout_box:
-        weight: -32
+        weight: 14
         enabled: false
       hs_carousel:
-        weight: -38
-        enabled: false
-      hs_clr_bnd:
-        weight: -43
-        enabled: true
-      hs_collection:
-        weight: -31
-        enabled: false
-      hs_gradient_hero:
-        weight: -30
-        enabled: false
-      hs_gradient_hero_slider:
-        weight: -29
+        weight: 10
         enabled: false
       hs_hero_image:
-        weight: -37
+        weight: 11
         enabled: false
       hs_postcard:
-        weight: -42
+        weight: 12
         enabled: true
       hs_priv_collection:
-        weight: -41
+        weight: 0
         enabled: true
       hs_priv_text_area:
-        weight: -45
+        weight: 16
+        enabled: true
+      hs_private_files:
+        weight: 14
         enabled: true
-      hs_row:
-        weight: -27
-        enabled: false
-      hs_spotlight:
-        weight: -26
-        enabled: false
       hs_sptlght_slder:
-        weight: -40
+        weight: 0
         enabled: true
-      hs_testimonial:
-        weight: -28
-        enabled: false
       hs_text_area:
-        weight: -35
-        enabled: false
-      hs_timeline:
-        weight: -39
-        enabled: true
-      hs_timeline_item:
-        weight: -25
+        weight: 14
         enabled: false
       hs_view:
-        weight: -34
-        enabled: false
-      hs_webform:
```

Comment:
In overridden config, try to include only the lines necessary for the the `hs_webform` paragraph removal. All other changes might be different from site to site and it's better to leave them out of the PR.

Reply:
Removed other changes.